paper.md

```diff
@@ -0,0 +1,400 @@
+
+Paper should include:
+
+- A summary describing the high-level functionality and purpose of the software for a diverse, non-specialist audience.
+- A clear Statement of Need that illustrates the research purpose of the software.
+- A list of key references, including to other software addressing related needs.
+- Mention (if applicable) a representative set of past or ongoing research projects using the software and recent scholarly publications enabled by it.
+- Acknowledgement of any financial support.
+
+-->
+
+<!-- RESOURCES
+
+- Empirical coding guidelines: https://github.com/devosoft/Empirical/wiki/Coding-Guidelines
+- Empirical README: https://github.com/devosoft/Empirical/blob/master/README.md
+
+-->
+
+# Summary
+<!-- SUMMARY
+
+JOSS: "Begin your paper with a summary of the high-level functionality of your software for a non-specialist reader. Avoid jargon in this section."
+
+i.e., What does this software do?
+-->
+
+# Summary
+
+Empirical is a C++ library designed to promote open science and facilitate the development of scientific software that is efficient, reliable, and easily distributable to researchers and non-experts alike.
+Specifically, the library sets out to fulfill the following goals:
+
+1. **Utility:** Empirical features a wide selection of helper tools to streamline common scientific computing tasks such as configuration, data management, random number generation, and mathematical manipulations. Documentation, examples, and project templates streamline developer onboarding.
+2. **Efficiency:** Empirical tools emphasize efficiency to make heavy scientific computing workloads tractable. Where possible, computation is moved to compile-time, heap allocation is avoided, and memory-locality is respected.
+3. **Reliability:** Scientific simulation and data analysis is meaningful insofar as it is correct. In conjunction with extensive unit testing, Empirical benefits from --- and provides --- sophisticated debug-mode instrumentation tools. These include audited memory management, drop-in replacements for standard C++ containers with safety checking, and self-documenting assertions.
+4. **Distributability:** Scientific software should be easily used by any researcher, student, or citizen scientist who wants to experiment, explore, or replicate studies. As such, Empirical is highly portable, uses common data formats, facilitates flexible runtime configuration, and simplifies compilation to a performant web app.
+
+# Statement of Need
+<!-- STATEMENT OF NEED
+JOSS: illustrates the research purpose of the software.
+JOSS reviewer checklist: "clearly state what problems the software is designed to solve and who the target audience is"
+-->
+
+Modern web-based interfaces give computational research the unique potential to embody open science objectives of making the scientific process more transparent with auditable and exensible code, clear and replicable methodologies, and production of accessible results [CITE osf.io].
```

Comment:
```
Modern web-based interfaces give computational research the unique potential to embody open science objectives of making the scientific process more transparent with auditable and exensible code, clear and replicable methodologies, and production of accessible results [CITE osf.io].
```

Don't think there's anything grammatically wrong here, but this would be more readable if broken up. E.g., period after "open science objectives" and make the rest into a new sentence.

Reply:
```suggestion
Modern web-based interfaces give computational research the unique potential to embody open science objectives.
They can make the scientific process more transparent with auditable and extensible code, clear and replicable methodologies, and production of accessible results [CITE osf.io].
```